packages/core/lib/services/blockchain/TransferService.js

```diff
@@ -55,7 +60,8 @@ export default class TransferService {
 		    case Blockchains.EOSIO: return transfer.transaction_id;
 		    case Blockchains.TRX: return transfer.txID;
 		    case Blockchains.ETH: return transfer.transactionHash;
-		    case Blockchains.BTC: return transfer.txid;
+            case Blockchains.BTC: return transfer.txid;
+            case Blockchains.PPY: return transfer;
```

Comment:
what exactly is returned here?

Reply:
`baseTransfer` in this case calls `PluginRepository.plugin` (accessing the respective blockchain plugin, in our case peerplays), and calls transfer on it. Most of these other chains have their transfer() returning an object which is then parsed. In our case, we're returning the hash directly (as mentioned in our convo), so I can just return the value of `transfer`.